docs/RLC-generation-Design.md

```diff
@@ -0,0 +1,57 @@
+# RLC generation Design
+
+## Basic Terms
+
+- **Cadl Files**: cloud service API definition written in [CADL](https://github.com/microsoft/cadl).
+- **Swagger Files**: REST api specification written in [OpenAPI](https://swagger.io/).
+- **CADL Program**: The model we get after CADL compiler has processed the cadl input.
```

Comment:
Don't quite understand this, is it the memory model for CADL input?

Reply:
yes